mwclient/client.py

```diff
@@ -612,6 +616,380 @@ def get_token(self, type, force=False, title=None):
+             ...     exists = site.create_user('SomeUser', 'Some password',
+             ...                               **my_extra_kwargs)
+             ... except mwclient.errors.CreateError as e:
+             ...     exists = e.code == 'userexists':
+             ...     if not exists:
+             ...         print('Can not create user: %s' % e)
+             ... if exists:
+             ...     user = site.get_user('SomeUser')
+
+         Args:
+             username (str): User name of the user to create
+             password (str): User password of the user to create
+             **kwargs (dict): Other required fields depending on your mediawiki conf
+                              eg: an email
+
+         Returns:
+             Bool (True) if user has beend created, else raise UserCreateError
+
+         Raises:
+             UserCreateError (mwclient.errors.UserCreateError): User can not be created
+                                                                for some reason
+             APIError (mwclient.errors.APIError): Other API errors
+        """
+
+        if 'createtoken' not in kwargs:
+            kwargs['createtoken'] = self.get_token('createaccount')
+        if 'retype' not in kwargs:
+            kwargs['retype'] = password
+        if 'continue' not in kwargs and 'createreturnurl' not in kwargs:
+            # should be great if API didn't require this...
+            kwargs['createreturnurl'] = '%s://%s' % (self.scheme, self.host)
+        info = self.post('createaccount', username=username, password=password, **kwargs)
+        if info['createaccount']['status'] == 'FAIL':
+            raise errors.UserCreateError(
+                info['createaccount'].get('messagecode'),
+                info['createaccount'].get('message'),
+                kwargs=kwargs
+            )
+        return True
+
+    def get_user(
+        self, username=None, userid=None,
+        usprop='registration|groups|blockinfo'
+    ):
+        """
+         Retrieves user informations
+         (registration, groups and blockinfo infos by default).
+
+             >>> try:
+             ...     site.get_user('SomeUser')
+             ... except mwclient.errors.UserNotFound:
+             ...     print('User seems to not exist')
+
+         Args:
+             username (str): User name to retrieve, OR
+             userid (int): User ID to retrieve
+
+         Returns:
+             Dictionary of the JSON user informations response
+
+         Raises:
+             ValueError: username and userid params are both not set or set
+             UserNotFound (mwclient.errors.UserNotFound): User has not been found
+             APIError (mwclient.errors.APIError): Other API errors
+        """
+        if (not username and not userid) or (username and userid):
+            raise ValueError('username OR userid are required')
+
+        kwargs = {
+            'list': 'users',
+            'usprop': usprop,
+        }
+        if username:
```

Comment:
```suggestion
        if username is not None:
```

It's generally safer to be more explicit. https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations